.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+
+updates:
+  - package-ecosystem: composer
+    directory: "/"
+    schedule:
+      interval: monthly
+      time: "06:30"
+    commit-message:
+      prefix: "- (php) "
+    target-branch: main
+    open-pull-requests-limit: 1
+
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: monthly
+      time: "06:30"
+    commit-message:
+      prefix: "- (js) "
+    target-branch: main
+    open-pull-requests-limit: 1
```

Comment:
Could you add timezones?
Also github actions and docker can be added

```yml
  - package-ecosystem: github-actions
    directory: "/"
    schedule:
      interval: monthly
      time: "06:30"
      timezone: "Europe/Warsaw"
    commit-message:
      prefix: "- (github actions) "
    target-branch: main
    open-pull-requests-limit: 1

  - package-ecosystem: docker
    directory: "/environment/dev/app"
    schedule:
      interval: monthly
      time: "06:30"
      timezone: "Europe/Warsaw"
    commit-message:
      prefix: "- (docker dev) "
    target-branch: main
    open-pull-requests-limit: 1
    ```

Reply:
added